common/logger/src/libra_logger.rs

```diff
@@ -125,10 +127,24 @@ impl LibraLoggerBuilder {
     }
 
     pub fn init(&mut self) {
-        self.build()
+        self.build();
     }
 
-    pub fn build(&mut self) {
+    pub fn build(&mut self) -> Arc<LibraLogger> {
```

Comment:
Init was the old name we used to "init" the logging system so in order to avoid changing everything in one of my last refactors I just created the alias. Usually builders return what they build so this is 1 part of that. So basically its something that still should be cleaned up  at some point